src/flattenPath.js

```diff
@@ -0,0 +1,26 @@
+import { pathOr, curry, merge } from 'ramda';
+
+/**
+ * Merges object under property path onto provided object.
```

Comment:
Let's borrow the description from [recompose API](https://github.com/acdlite/recompose/blob/master/docs/API.md#flattenprop)

`Flattens a property path so that its fields are spread out into the provided object.`